Library/ViewModels/PledgeViewModel.swift

```diff
@@ -665,38 +667,44 @@ private func requiresSCA(_ envelope: StripeSCARequiring) -> Bool {
   return envelope.requiresSCAFlow
 }
 
-private func attributedConfirmationString(with project: Project) -> NSAttributedString? {
-  let string = Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_on_project_deadline(
-    project_deadline: Format.date(
-      secondsInUTC: project.dates.deadline,
-      template: "MMMM d, yyyy"
-    )
+private func attributedConfirmationString(with project: Project, pledgeTotal: Double) -> NSAttributedString? {
+  var fullString = ""
+  let date = Format.date(
+    secondsInUTC: project.dates.deadline,
+    template: "MMMM d, yyyy"
   )
+  let pledgeTotal = Format.currency(pledgeTotal, country: project.country)
 
-  guard let attributedString = try? NSMutableAttributedString(
-    data: Data(string.utf8),
-    options: [
-      .documentType: NSAttributedString.DocumentType.html,
-      .characterEncoding: String.Encoding.utf8.rawValue
-    ],
-    documentAttributes: nil
-  ) else { return nil }
+  if project.stats.currentCurrency == project.stats.currency {
```

Comment:
To make this function a bit smaller, we could extract this to a new function that returns a String based on the currencies.